packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -487,23 +493,18 @@ export function generateGrid(
                 ${
                     showActionsColumn
                         ? `{
-                        field: "actions",
-                        headerName: "",
-                        sortable: false,
-                        filterable: false,
-                        type: "actions",
-                        align: "right",
+                        ${actionsColumnProps.join(",\n")},
```

Comment:
you might consider using `tsCodeRecordToString({
    field: "\"actions\"",
    headerName: "\"\"",
    "sortable: false",
//   ....
})`